### ISSUE TITLE
Question about pixel-space loss computation in Stage 2 training

### ISSUE THREAD

**Issue by bjkim95** [OpenRobotLab/Aether#15]
Hi, thank you for the great work!

I have a question regarding the Stage 2 training process, specifically how the pixel-space loss is computed.
From my understanding, the losses are applied after decoding the latents into the pixel space.

Is this loss computed by first obtaining z_0 from the predicted velocity and then decoding it to x_0?
If that’s the case, wouldn’t x_0 become quite blurry for large diffusion timesteps?

Also, do you sample all diffusion timesteps randomly for this stage, as you did during Stage 1 training?

Thank you in advance!

**Comment by HaoyiZhu** [comment]
Hi, thanks for your interest!

We try this because we find that only 3-4 denoising steps can achieve the geometry modalities for reconstruction tasks. Actually, the training steps of this stage are much fewer than the standard stage. Also, we admit that this additional stage does *not* provide much performance improvement, especially for generation. 

We believe better strategies can be explored.

**Comment by bjkim95** [comment]
Thank you for your kind answer!
